x-pack/plugins/secops/server/lib/overview/query.dsl.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { createQueryFilterClauses } from '../../utils/build_query';
+import { RequestBasicOptions } from '../framework';
+
+export const buildOverviewNetworkQuery = ({
+  filterQuery,
+  timerange: { from, to },
+  sourceConfiguration: {
+    fields: { timestamp },
+    auditbeatAlias,
+    logAlias,
+    packetbeatAlias,
+  },
+}: RequestBasicOptions) => {
+  const filter = [
+    ...createQueryFilterClauses(filterQuery),
+    {
+      range: {
+        [timestamp]: {
+          gte: from,
+          lte: to,
+        },
+      },
+    },
+  ];
+
+  const dslQuery = {
+    allowNoIndices: true,
+    index: [auditbeatAlias, logAlias, packetbeatAlias],
+    ignoreUnavailable: true,
+    body: {
+      aggregations: {
+        unique_flow_count: {
+          filter: {
+            term: { type: 'flow' },
+          },
+        },
+        unique_dns_count: {
+          filter: {
+            term: { type: 'dns' },
+          },
+        },
+        unique_suricata_count: {
+          filter: {
+            term: { 'service.type': 'suricata' },
+          },
+        },
+        unique_zeek_count: {
+          filter: {
+            term: { 'service.type': 'zeek' },
```

Comment:
Rather than using `service.type`, it has been recommended we use `event.module` since `service.type` gets "tacked" on somewhere along the pipeline and it is more direct to use event.

Plus, it will go nicely with the word `event.dataset` below ;-)